assets/questions/findMostCommonCharacter.js

```diff
@@ -102,31 +102,5 @@ class AuxiliaryCode(object):
       ]
     }],
     performanceTests: []
-  }, {
-    instructions: [{
-      content: [
-        'Now, make sure that your code works for ',
-        'different string encodings. ',
-        'What if the provided string is unicode, rather than ASCII?'
-      ].join(''),
-      type: 'text'
-    }],
-    prerequisiteSkills: ['Arrays', 'Strings', 'Hash Maps'],
-    acquiredSkills: ['String Manipulation'],
-    inputFunctionName: null,
-    outputFunctionName: null,
-    mainFunctionName: 'findMostCommonCharacter',
-    correctnessTests: [{
-      input: '\u0041\u0042\u0043\u0041',
-      allowedOutputs: ['A'],
-      tag: 'the general case'
-    }],
-    buggyOutputTests: [],
-    performanceTests: [{
```

Comment:
I think dropping this will drop performance tests from the entire problem. Can you move this further down in the problem?